packages/go/index.ts

```diff
@@ -100,7 +94,7 @@ export async function build({
   workPath,
   meta = {},
 }: BuildOptions) {
-  if (process.env.GIT_CREDENTIALS && !meta.isDev) {
+  if (process.env.GIT_CREDENTIALS) {
```

Comment:
Are you sure this should happen in `vc dev`?

Reply:
It doesn't look like `vc dev` calls `build`. In `startDevServer`, there is no mention of `initPrivateGit`. I believe this is what already happens because `isDev` is never true when `build` is called, but if I'm mistaken we can dig deeper into this.

Reply:
Hmm, might be left over from before startDevServer was implemented 👍